master/buildbot/steps/source/git.py

```diff
@@ -770,3 +661,87 @@ def _doTag(self):
 
         ret = yield self._dovccmd(cmd)
         return ret
+
+
+class GitCommit(buildstep.BuildStep, GitStepMixin, CompositeStepMixin):
+
+    description = None
+    descriptionDone = None
+    descriptionSuffix = None
+
+    name = 'gitcommit'
+    renderables = ['files', 'messages']
+
+    def __init__(self, workdir=None, files=None, messages=None, env=None,
+                 timeout=20 * 60, logEnviron=True,
+                 config=None, **kwargs):
+
+        self.workdir = workdir
+        self.messages = messages
+        self.files = files
+        self.env = env
+        self.timeout = timeout
+        self.logEnviron = logEnviron
+        self.config = config
+        # These attributes are not required for a git add command
```

Comment:
I think this comment could be reworded to something like: "The repourl, sshPrivateKey and sshHostKey attributes are required by GitStepMixin, but aren't needed by git add and commit operations"